qcdb/util/regex.py

```diff
@@ -30,7 +30,7 @@
 
 yes = re.compile(r'^(yes|true|on|1)', re.IGNORECASE)
 no = re.compile(r'^(no|false|off|0)', re.IGNORECASE)
-der0th = re.compile(r'^(0|none|energy)', re.IGNORECASE)
+der0th = re.compile(r'^(0|none|energy|property)', re.IGNORECASE)
```

Comment:
not the best, but then properties throws a wrench into many things. we'll see how this shakes out.